tree/src/test/java/org/infinispan/api/tree/NodeMoveAPIPessimisticTest.java

```diff
@@ -22,4 +22,11 @@ protected ConfigurationBuilder createConfigurationBuilder() {
             .transaction().lockingMode(LockingMode.PESSIMISTIC);
       return cb;
    }
+
+   @Test(enabled = false, description = "The test does not work on pessimistic RR caches, " +
+         "because moveCtoB reads in the root node into context. Then, the move operation does not check " +
+         "if the entry was not changed after lock (no pessimistic WSC) and the move can be executed.")
```

Comment:
The test was passing by coincidence IIRC, but generally, the `move` did not work in all cases.